ci/playbooks/deploy_edpm.yaml

```diff
@@ -0,0 +1,79 @@
+---
+- hosts: controller
+  vars:
+    install_yamls_basedir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls"
+  tasks:
+    - name: Install libguestfs-tools-c package
+      become: true
+      package:
+        name: libguestfs-tools-c
+        state: present
+
+    - name: Run make edpm_compute
+      community.general.make:
+        target: edpm_compute
+        chdir: "{{ install_yamls_basedir }}/devsetup"
+        params:
+          OUT: "{{ install_yamls_basedir }}/out"
+          EDPM_COMPUTE_SUFFIX: 0
+
+    - name: Sleep for 60 secs to get compute node up
+      ansible.builtin.shell: |
+        sleep 60;
+
+    - name: Run make edpm_compute_repos
+      community.general.make:
+        target: edpm_compute_repos
+        chdir: "{{ install_yamls_basedir }}/devsetup"
+        params:
+          EDPM_COMPUTE_SUFFIX: 0
+
+    - name: Run EDPM Deploy
+      community.general.make:
+        target: edpm_deploy
+        chdir: "{{ install_yamls_basedir }}"
+        params:
+          OUT: "{{ install_yamls_basedir }}/out"
+          DATAPLANE_CHRONY_NTP_SERVER:  pool.ntp.org
+
+    - name: Get info about dataplane node
+      ansible.builtin.shell: |
+        sleep 60;
+        oc get openstackdataplane
+        oc get openstackdataplanerole
+        oc get openstackdataplanenode
+        oc get pods | grep edpm
+      ignore_errors: true
+
+    - name: Get dataplane-deployment-run pod name
+      register: edpm_pod
+      ansible.builtin.shell:
+        cmd: oc get pods -o name | grep -m1  dataplane-deployment-run-openstack
+        executable: /bin/bash
+      until: edpm_pod.rc == 0
+      retries: 150
+      delay: 50
+
+    - name: Wait for dataplane-deployment-run pod name
+      register: deploy_status
+      ansible.builtin.shell:
+        cmd: "oc wait --timeout=0 --for=jsonpath='{.status.phase}'=Succeeded {{ edpm_pod.stdout |trim }}"
+      until: deploy_status.rc == 0
+      retries: 100
+      delay: 20
+
+    - name: Get the logs of EDPM Play
```

Comment:
You could get the logs of all EDPM plays [1] not just the `dataplane-deployment-run-openstack-*` one.

[1]  https://paste.opendev.org/show/bM8DkvYsfMaEX0vFhqMI/

Reply:
For example...

`for POD in $(oc get pods -o name | egrep "dataplane|edpm"); do echo $POD; oc logs $POD; done`

If one of the many playbooks fail, we'll want their logs.

Reply:
Done